Cargo.toml

```diff
@@ -2,11 +2,11 @@
 name = "battleship"
 version = "0.1.0"
 authors = [
-    "Dennis Altenhoff <daltenhoff@uos.de>",
-    "Tim Bohne <tbohne@uos.de>",
-    "Svantje Jung <svjung@uos.de>",
-    "Famke Lamberti <flamberti@uos.de>"
+    "Dennis Altenhoff <daltenhoff@uos.de> ",
```

Comment:
Where? I don't see anything missing. Spaces at the beginning of the other names for formatting purpose on runtime.